api.planx.uk/send/uniform.ts

```diff
@@ -414,13 +414,16 @@ const createUniformApplicationAuditRecord = async ({
   localAuthority: string,
   submissionDetails: UniformSubmissionResponse,
   }): Promise<UniformApplication> => {
+  const xml = await _admin.generateOneAppXML(payload?.sessionId);
```

Comment:
Regenerating this here shouldn't be too expensive and it's an async action so time is not of the essence. The alternative of generating it once and passing along to where it's required felt like quite a lot of code pollution for a small fix.